### PR TITLE
Correct .magento.app.yaml description for "name" property

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

In the _Cloud Guide_, the application configuration instructions mention that you can configure multiple applications in Magento Cloud projects. However,  Cloud Pro projects do not support multiple application installation.

Removed incorrect information about the `name` property in the `.magento.app.yaml` file. 

## Affected DevDocs pages

https://devdocs.magento.com/cloud/project/project-conf-files_magento-app.html#name

### CODE REVIEW

File: src/cloud/project/project-conf-files_magento-app.md [hunk 1]
```diff
@@ -18,12 +18,10 @@ Use the following properties to build your application configuration file. The `
 
 ### `name`
 
-{{site.data.var.ee}} supports multiple applications in a project, so you need a unique name that identifies the application in the project. You must use lower case alphanumeric characters, such as `a` to `z` and `0` to `9` for the name.
-
-The name is used in the [`routes.yaml`]({{ site.baseurl }}/cloud/project/project-conf-files_routes.html) file to define the HTTP upstream (by default, `php:http`). For example, if the value of `name` is `app`, you must use `app:http` in the upstream field. You can also use this name in multi-application relationships.
+The `name` property provides the application name used in the [`routes.yaml`]({{ site.baseurl }}/cloud/project/project-conf-files_routes.html) file to define the HTTP upstream (by default, `php:http`). For example, if the value of `name` is `app`, you must use `app:http` in the upstream field. You can also use this name in multi-application relationships.
```
Comment (NadiyaS) [inline src/cloud/project/project-conf-files_magento-app.md:21]: by default it is `mymagento:http`

Comment (NadiyaS) [reply src/cloud/project/project-conf-files_magento-app.md:21]: And I even not sure that we need to describe how to change it as we do not support having several application - renaming does not have any sense

Comment (meker12) [inline src/cloud/project/project-conf-files_magento-app.md:21]:
```suggestion
The `name` property provides the application name used in the [`routes.yaml`]({{ site.baseurl }}/cloud/project/project-conf-files_routes.html) file to define the HTTP upstream (by default, `mymagento:http`). For example, if the value of `name` is `app`, you must use `app:http` in the upstream field. You can also use this name in multi-application relationships.
```

File: src/cloud/project/project-conf-files_magento-app.md [hunk 2]
```diff
@@ -18,12 +18,10 @@ Use the following properties to build your application configuration file. The `
 
 ### `name`
 
-{{site.data.var.ee}} supports multiple applications in a project, so you need a unique name that identifies the application in the project. You must use lower case alphanumeric characters, such as `a` to `z` and `0` to `9` for the name.
-
-The name is used in the [`routes.yaml`]({{ site.baseurl }}/cloud/project/project-conf-files_routes.html) file to define the HTTP upstream (by default, `php:http`). For example, if the value of `name` is `app`, you must use `app:http` in the upstream field. You can also use this name in multi-application relationships.
+The `name` property provides the application name used in the [`routes.yaml`]({{ site.baseurl }}/cloud/project/project-conf-files_routes.html) file to define the HTTP upstream (by default, `mymagento:http`). For example, if the value of `name` is `app`, you must use `app:http` in the upstream field. You can also use this name in multi-application relationships.
```
Comment (NadiyaS) [inline src/cloud/project/project-conf-files_magento-app.md:21]: > You can also use this name in multi-application relationships.

For me it looks like official approval of using multiple application on cloud environment.
I would remove it.

Comment (meker12) [inline src/cloud/project/project-conf-files_magento-app.md:21]:
```suggestion
The `name` property provides the application name used in the [`routes.yaml`]({{ site.baseurl }}/cloud/project/project-conf-files_routes.html) file to define the HTTP upstream (by default, `mymagento:http`). For example, if the value of `name` is `app`, you must use `app:http` in the upstream field.
```

**Pull request merged**